tareas/clase-5/index-mio.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html lang="es">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Clase 5</title>
+</head>
+<body>
+    
+        <h1>Calculador de sueldos</h1>
+    
+    <form>
+    <label> Ingrese su salario anual:</label>
+        <input type="text" id="ingresar-salario-anual"/>
+            <button type="button" id="calcular-salario-mensual"> Calcular </button>
+                    <label>Su salario mensual es: </label>
+                        <input type="text" id="salario-mensual" disabled/>
+                            <button type="reset" > Limpiar </button>                   
+    <br><br>
+
+    <label> Ingrese su salario mensual:</label>
+         <input type="text" id=ingresar-salario-mensual/>
```

Comment:
```suggestion
         <input type="text" id="ingresar-salario-mensual"/>
```
Faltaban las comillas.